### PR TITLE
deprecate ramulator; replace with dramsim3 files

### PR DESCRIPTION
**Pull request by tommydcjung**

### CODE REVIEW

**Comment by tommydcjung** [comment]
https://github.com/bespoke-silicon-group/basejump_stl

**Comment by tommydcjung** [comment]
https://github.com/bespoke-silicon-group/basejump_stl/pull/241

File: machines/4x4_dmc/Makefile.machine.include [hunk 1]
```diff
@@ -3,6 +3,7 @@ BSG_MACHINE_GLOBAL_Y                  = 5
 BSG_MACHINE_VCACHE_SET                = 128
 BSG_MACHINE_VCACHE_WAY                = 8
 BSG_MACHINE_VCACHE_BLOCK_SIZE_WORDS   = 8
+BSG_MACHINE_VCACHE_DMA_DATA_WIDTH     = 32
```
Comment (drichmond) [inline machines/4x4_dmc/Makefile.machine.include]: Is there a reason this wasn't added to 9x15? Are we not modifying 9x15?

File: machines/Makefile [hunk 2]
```diff
@@ -15,16 +15,18 @@ VCS_INCLUDES += $(foreach inc,$(VINCLUDES),+incdir+"$(inc)")
 VCS_DEFINES  += $(foreach def,$(VDEFINES),+define+"$(def)")
 VCS_SOURCES  += $(VHEADERS) $(VSOURCES) 
 VCS_FLAGS    += +v2k -sverilog -full64 -timescale=1ps/1ps \
-	 	+lint=all,noSVA-UA,noSVA-NSVU,noVCDE,noNS
+	 	+lint=all,noSVA-UA,noSVA-NSVU,noVCDE,noNS -top spmd_testbench
 
-VCS_CFLAGS   =  -cpp clang++
+VCS_CFLAGS   =  -cpp g++
```
Comment (drichmond) [inline machines/Makefile]: Just generally curious why clang was necessary at one point. Not relevant to feedback on this PR. Did it work?

File: machines/Makefile [hunk 3]
```diff
@@ -15,16 +15,18 @@ VCS_INCLUDES += $(foreach inc,$(VINCLUDES),+incdir+"$(inc)")
...
 VCS_CFLAGS   += -CFLAGS "-std=c++11 -g -Wall"
 VCS_CFLAGS   += -CFLAGS "-fPIC"
-VCS_CFLAGS   += -CFLAGS "-I$(BSG_MANYCORE_DIR)/imports/ramulator/src"
-VCS_CFLAGS   += -CFLAGS "-DRAMULATOR"
-VCS_CFLAGS   += -CFLAGS "-DHBM_CONFIG_PATH=$(BSG_MANYCORE_DIR)/imports/ramulator/configs/HBM-config.cfg"
+VCS_CFLAGS 	 += -CFLAGS "-I$(BASEJUMP_STL_DIR)/imports/DRAMSim3/src"
```
Comment (drichmond) [inline machines/Makefile]: Pedantic, I know, but there's an extra space here

File: machines/4x4_dmc/Makefile.machine.include (hunk 1 above)
Comment (tommydcjung) [reply machines/4x4_dmc/Makefile.machine.include]: 9x15 will be dropped.

File: machines/Makefile (hunk 2 above)
Comment (tommydcjung) [reply machines/Makefile]: ramulator had some c++ stuff that required it.

**Pull request merged**